app/src/main/java/com/vuongnq14798/vuongmp3/service/MediaPlayerService.kt

```diff
@@ -0,0 +1,136 @@
+package com.vuongnq14798.vuongmp3.service
+
+import android.app.Service
+import android.content.Context
+import android.content.Intent
+import android.media.MediaPlayer
+import android.os.Binder
+import android.os.IBinder
+import android.widget.Toast
+import com.vuongnq14798.vuongmp3.R
+import com.vuongnq14798.vuongmp3.data.model.Track
+import com.vuongnq14798.vuongmp3.mediaplayer.MediaPlayerManager
+import com.vuongnq14798.vuongmp3.util.LoopType
+import com.vuongnq14798.vuongmp3.util.StateType
+
+class MediaPlayerService : Service(), MediaPlayer.OnPreparedListener, MediaPlayer.OnErrorListener,
+    MediaPlayer.OnCompletionListener, ServiceInterface {
```

Comment:
conventions
https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting
For multiple interfaces, the superclass constructor call should be located first and then each interface should be located in a different line: